changelog.txt

```diff
@@ -1,11 +1,12 @@
 V.Next
 ----------
+- [PATCH] Port eSTS telemetry layer to common-java (#1328)
+- [PATCH] Add common-java. Migrate Logger. (#1300)
```

Comment:
Didn't this already merge into dev?

Reply:
I (unknowingly) moved it up during merge.